README.adoc

```diff
@@ -185,6 +185,10 @@ $ for i in "$(ls cpu-*)";
 
 ----
 
+=== Chronicle Tune integration
+
+https://chronicle.software/tune/[Chronicle Tune] is software that automatically configures your OS for optimal performance. Chronicle Tune can be configured to isolate cores for running low-latency workloads. If Chronicle Tune is actively configured on the target system then Java-Thread-Affinity will automatically load the Tune configuration and record which cores have been isolated. In order to ensure the best performance and the least surprises out of the box Affinity will only consider locking workloads to cores isolated via Chronicle Tune. If Chronicle Tune is installed but no cores are isolated then Affinity will consider all cores as potential lock candidates.
```

Comment:
pls remove the word "automatically" from the first sentence

Reply:
Suggest add "also" to " Chronicle Tune can **also** be configured"....

Reply:
Thanks - that's updated